specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2022-09-30-privatepreview/examples/AzureADAdministratorDelete.json

```diff
@@ -8,7 +8,12 @@
   },
   "responses": {
     "200": {},
-    "202": {},
+    "202": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/providers/Microsoft.DBforMySQL/locations/southeastasia/azureAsyncOperation/e276a03a-1770-4549-86f5-0edffae8997c?api-version=2022-09-30-privatepreview",
+        "Location": "https://management.azure.com/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/providers/Microsoft.DBforMySQL/locations/southeastasia/operationResults/e276a03a-1770-4549-86f5-0edffae8997c?api-version=2022-09-30-privatepreview"
```

Comment:
>Location [](http://example.com/codeflow?start=9&length=8)

Since you include both headers, I suggest you might want to add the x-ms-final-state-via attributes as a hint to everyone which kind of header you recommend people to use.